2 - RTB/Documentazione interna/Verbali/24-02-04/24-02-04.typ

```diff
@@ -0,0 +1,37 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "04/02/24",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Oseliero Antonio",
+  ),
+  timeStart: "15:00",
+  timeEnd: "15:40",
+);
+
+= Ordine del giorno
+- Discussione riguardo esito RTB con professor Cardin;
+- Discussione riguardo stato di avanzamento e RTB con professor Vardanega;
+- Pianificazione sprint, ruoli e prossimi obbiettivi
+
+== Valutazione esito RTB
+Il team ha aperto la riunione discutendo l'esito della prima parte di RTB svolta con il professor Cardin. Si è discusso in particolare delle criticità rilevate nel documento di `Analisi dei Requisiti`, che possono essere riassunte in:
+- alcuni errori nella modellazione;
+- poca specificità degli UC;
+- assente tracciamento casi d'uso/requisiti.
+
+== Pianificazione seconda parte RTB
+Il team procede quindi esaminando la situazione corrente e settando degli obbiettivi "a lungo termine" da concludere come prerequisito per la seconda parte del RTB con il professor Vardanega. In particolare si è visto:
+- aggiornamento in seguito al big-bang del `Piano di Progetto`;
+- modifica di alcune metriche del `Piano di Qualifica` non compatibili con il modello di sviluppo scelto dal team;
+- necessità di preparare le slide per l'incontro con il professore;
+- priorità data a: `Analisi dei Requisiti`, `Piano di Qualifica` e `Norme di Progetto`.
+
+== Pianificazione <pianificazione>
+
+La pianificazione per lo sprint in corso ha visto definiti:
+- suddivisione ruoli e task;
+- revisione suddivisione ruoli per prolungamento del RTB oltre le attese;
+- tolti i programmatori e reinseriti gli analisti in vista del lavoro sull' `Analisi dei Requisiti`.
```

Comment:
metterei "rimozione dei Programmatori e reinserimento degli Analisti tra i ruoli dello sprint in vista...."